x-pack/plugins/canvas/canvas_plugin_src/functions/common/revealImage.ts

```diff
@@ -75,8 +74,8 @@ export function revealImage(): ExpressionFunctionDefinition<
         value: {
           percent,
           ...args,
-          image: resolveWithMissingImage(args.image, elasticOutline),
-          emptyImage: resolveWithMissingImage(args.emptyImage),
+          image: resolveWithMissingImage(args.image, elasticOutline) as string,
+          emptyImage: resolveWithMissingImage(args.emptyImage) as string,
```

Comment:
probably `resolveWithMissingImage` should return strings only instead of `string | null`? I left the current behavior since it's to be desirable according to the unit tests.

Reply:
Agree, that should be cleaned up.